x-pack/plugins/index_lifecycle_management/public/application/services/api_errors.ts

```diff
@@ -4,10 +4,13 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { IHttpFetchError } from 'src/core/public/http';
```

Comment:
I think this can be imported from `src/core/public` and then you can remove the `eslint-disable-next-line`